js/src/lib/interfaces/Types.ts

```diff
@@ -36,6 +36,8 @@ export enum PipelineType {
 	"image-segmentation"                                      = "Image Segmentation",
 	"text-to-image"                                           = "Text-to-Image",
 	"image-to-text"                                           = "Image-to-Text",
+	"image-to-image"                                          = "Image-to-Image",
+	"image-generation"										  = "Image Generation",
```

Comment:
There are some weird indentation issues here and in a couple of other sections below

Reply:
I blame GitHub CoPilot!! Haha I guess it inserted tabs instead of spaces, and it showed it was aligned in VSCode but looks different here. Should be taken care of now 🤗

Reply:
Thanks!!